### DISCUSSION TITLE
Why Quartz expression disappears when workflow starts execution?

### DISCUSSION THREAD

**Discussion by rairafaqatali** [elsa-workflows/elsa-core#1571] [Q&A]
Hi @sfmskywalker 
I have created a workflow in which I have provided a cron expression. Cron will work fine after the expression meets the criteria.
I have added a database to store quartz expression.

But I have identified an issue in the **QRTZ_TRIGGERS** table, this table has entry only when the workflow is not executing, but when the workflow starts its execution, entry is removed from this table and inserted back to this table once the workflow completes its execution.

Can you please let me know how can I persist the value here in this table?

**Comment by sfmskywalker** [comment]
The tables prefixed with `QRTZ_` are all used by the Quartz.NET library - Elsa doesn't work with these directly.

**Reply by rairafaqatali** [reply]
@sfmskywalker How we relate Elsa with Quartz?

When Elsa starts execution of the scheduled workflow, entry from the Quartz table gets removed.

**Reply by sfmskywalker** [reply]
Elsa uses the Quartz.NET API to schedule work, but it does not deal with the Quartz tables directly. If entries get removed, it is Quartz.NET that is doing so. Perhaps it performs some cleanup.

You mentioned this is an issue. Does that mean the timer activity doesn't work?
